### PR TITLE
Input Components Props Misc

### PR DESCRIPTION
**Pull request by philson-philip**
Fixes: #173 
@goutham-subramanyam _a I have changed the `helpText` to `infoText` and added a `helpText` prop to display help message in the input components. Can you check it?

### CODE REVIEW

File: lib/components/DateInput.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ const DateInput = React.forwardRef(
       label,
       labelProps,
       required = false,
-      helpText,
+      infoText,
```
Comment (goutham-subramanyam) [inline lib/components/DateInput.js:16]:
```suggestion
      infoText = "",
```

File: lib/components/Select.js [hunk 2]
```diff
@@ -15,25 +15,28 @@ class Select extends Component {
       value,
       onChange,
       leadingIcon = null,
+      infoText = "",
+      infoTextPosition = "right",
```
Comment (goutham-subramanyam) [inline lib/components/Select.js:19]:
```suggestion
      infoText = "",
```

File: lib/components/Select.js [hunk 3]
```diff
@@ -15,25 +15,28 @@ class Select extends Component {
...
       helpText = "",
-      helpTextPosition = "right",
       suffix = "",
       prefix = "",
```
Comment (goutham-subramanyam) [inline lib/components/Select.js:22]:
```suggestion
```

File: lib/components/TimeInput.js [hunk 4]
```diff
@@ -16,7 +16,8 @@ export default function TimeInput(props) {
     useAmPm = true,
     minuteStep = 1,
     labelProps,
-    helpText,
+    infoText,
```
Comment (goutham-subramanyam) [inline lib/components/TimeInput.js:19]:
```suggestion
    infoText = "",
```

File: lib/components/Textarea.js [hunk 5]
```diff
@@ -13,7 +13,8 @@ class Textarea extends Component {
       error,
       value,
       onChange,
-      helpText,
+      infoText,
```
Comment (goutham-subramanyam) [inline lib/components/Textarea.js:16]:
```suggestion
      infoText = "",
```

File: lib/components/Slider.js [hunk 6]
```diff
@@ -8,33 +8,31 @@ const Slider = ({
   label,
   labelProps,
   required,
-  helpText,
+  infoText,
```
Comment (goutham-subramanyam) [inline lib/components/Slider.js:11]:
```suggestion
  infoText = "",
```

**Review by goutham-subramanyam — CHANGES_REQUESTED** [review]
@philson-philip _a

File: lib/components/Select.js (hunk 2 above)
Comment (philson-philip) [reply lib/components/Select.js:19]: @goutham-subramanyam We are using these props in NeetoDesk. Should we remove it?

**Comment by philson-philip** [comment]
@goutham-subramanyam _a

**Review by goutham-subramanyam — APPROVED** [review]
LGTM.

**Pull request merged**